examples/experimental/persistent-volume-provisioning/README.md

```diff
@@ -81,6 +81,28 @@ parameters:
 * `type`: `pd-standard` or `pd-ssd`. Default: `pd-ssd`
 * `zone`: GCE zone. If not specified, a random zone in the same region as controller-manager will be chosen.
 
+#### GLUSTERFS
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: StorageClass
+metadata:
+  name: slow
```

Comment:
any reason why "name: slow"? If this is an example, we probably want to choose a word other than _slow_